### PR TITLE
sets excon timeout

### PR DESCRIPTION
**Pull request by dougperez**
Still need to test this.

We are seeing alot of timeout for pwg lets try 45 sec.

### CODE REVIEW

File: lib/conduit/pwg/actions/base.rb [hunk 1]
```diff
@@ -23,5 +23,9 @@ def perform
     rescue Excon::Errors::Timeout
       raise(Conduit::TimeOut, 'Connection timeout error.')
     end
+
+    def connection
+      @excon ||= Excon.new(remote_url, timeout: 45.seconds)
```
Comment (pedromartinez) [inline lib/conduit/pwg/actions/base.rb]: It looks like we may be setting other things in params, like header records. Maybe we should merge this override onto params?

https://github.com/conduit/conduit/blob/347e7b81bb6cef84eebdf8076114729bd7c6c63f/lib/conduit/core/connection.rb#L81

Comment (dougperez) [reply lib/conduit/pwg/actions/base.rb]: Do you know what I can override for params? Or is that something that doesn't exist yet?

Comment (pedromartinez) [reply lib/conduit/pwg/actions/base.rb]: Nevermind, looking at that code again, it doesn't even look like we're using the params passed into the connection object. It's a private method with optional args and we're calling it with no args passed.

This one looks fine to me.

**Comment by pedromartinez** [comment]
Needs a version bump though.

**Pull request closed without merging**